src/Symfony/Component/Notifier/Bridge/Amazon/README.md

```diff
@@ -0,0 +1,40 @@
+Amazon Notifier
+=============
+
+Provides [Amazon SNS](https://aws.amazon.com/de/sns/) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```ini
```

Comment:
Sorry can you please remove the ini and add an environment variable assignment back?

Thanks